package.json

```diff
@@ -36,7 +36,7 @@
   "homepage": "https://github.com/Skyscanner/backpack-ios#readme",
   "devDependencies": {
     "bpk-svgs": "^6.0.2",
-    "bpk-tokens": "^27.2.1",
+    "bpk-tokens": "^27.3.1-beta.11",
```

Comment:
Are we planning to release like this or will we wait for a stable release of `bpk-tokens`?

Reply:
I think we should wait for a stable release. We have pretty much everything we need:

- [x] Tokens source of truth
- [x] iOS (this PR)
- [x] RN
- [x] Android (https://github.com/Skyscanner/backpack-android/pull/128/files)

So once iOS and Android are merged we can do quick follow-up PRs to bump to stable tokens and release all repos at once.

Reply:
Sounds good then